dask/array/tests/test_cupy_random.py

```diff
@@ -58,6 +58,35 @@ def rnd_test(func, *args, **kwargs):
     rnd_test(rs.standard_t, 2, size=5, chunks=3)
 
 
+def test_random_all_Generator():
+    def rnd_test(func, *args, **kwargs):
+        a = func(*args, **kwargs)
+        assert type(a._meta) == cupy.ndarray
+        assert_eq(a, a)  # Check that _meta and computed arrays match types
+
+    cupy_generator = cupy.random.default_rng()
+    rs = da.random.default_rng(seed=cupy_generator)
+
+    rnd_test(rs.beta, 1, 2, size=5, chunks=3)
+    rnd_test(rs.binomial, 10, 0.5, size=5, chunks=3)
+    rnd_test(rs.chisquare, 1, size=5, chunks=3)
+    rnd_test(rs.exponential, 1, size=5, chunks=3)
+    rnd_test(rs.f, 1, 2, size=5, chunks=3)
+    rnd_test(rs.gamma, 5, 1, size=5, chunks=3)
+    rnd_test(rs.geometric, 1, size=5, chunks=3)
+    rnd_test(rs.hypergeometric, 1, 2, 3, size=5, chunks=3)
+    rnd_test(rs.integers, 1, high=10, size=5, chunks=3)
+    rnd_test(rs.logseries, 0.5, size=5, chunks=3)
+    rnd_test(rs.poisson, 1, size=5, chunks=3)
+
+    rnd_test(rs.power, 1, size=5, chunks=3)
+    rnd_test(rs.random, size=5, chunks=3)
+
+    rnd_test(rs.standard_exponential, size=5, chunks=3)
+    rnd_test(rs.standard_gamma, 2, size=5, chunks=3)
+    rnd_test(rs.standard_normal, size=5, chunks=3)
+
+
 @pytest.mark.parametrize("shape", [(2, 3), (2, 3, 4), (2, 3, 4, 2)])
 def test_random_shapes(shape):
```

Comment:
Should we include a multiarray test given the multiarray handling above using `*args`, `**kwargs`, and maybe some mix of both?

Reply:
I am not sure what you mean but added tests with different sizes